acceptance/tests/base/external_resources_test.rb

```diff
@@ -1,9 +1,11 @@
 test_name 'External Resources Test' do
   step 'Verify EPEL resources are up and available' do
     def epel_url_test(el_version)
-      url = "#{options[:epel_url]}/epel-release-latest-#{el_version}.noarch.rpm"
+      url_base = options[:epel_url]
+      url_base = options[:epel_url_archive] if el_version == 5
```

Comment:
Would it be nicer to have this assignment as `url_base = el_version == 5 ? options[:epel_url_archive] : options[:epel_url]`?